webapp/store/actions/socketActions.js

```diff
@@ -4,13 +4,15 @@ import { getClient } from '@bpanel/bpanel-utils';
 const client = getClient();
 
 export function connectSocket() {
-  return {
-    type: CONNECT_SOCKET,
-    bsock: {
-      host: client.host ? client.host : 'localhost',
-      port: 8000,
-      namespace: client.id
-    }
+  return (dispatch, getState) => {
+    dispatch({
+      type: CONNECT_SOCKET,
+      bsock: {
+        host: client.host ? client.host : 'localhost',
+        port: getState().app.socketPort,
```

Comment:
I know this isn't great. Ideally this would connect through the same port as the app, but since bsock's server can't really attach to the express server right now, we kind of have to hard code it like this (I added it to the app state now at least rather than just hardcoded here so at least it can be shared). Once we migrate to using bweb for the bPanel server though, this shouldn't be necessary anymore.

Reply:
Ok that sounds good to me 👍